mava/wrappers/environment_loop_wrappers.py

```diff
@@ -254,6 +255,7 @@ def _compute_episode_statistics(
         self._episode_length_stats.push(episode_steps)
         self._episode_return_stats.push(mean_episode_return)
         self._steps_per_second_stats.push(steps_per_second)
+        self._running_statistics["sum_episode_return"] = sum_episode_return
```

Comment:
Should we potentially treat this in the same way we are current treating the episode return which is the mean over all agent returns? That way we would have `Raw Sum Episode Return`, `Mean Sum Episode Return` etc logged. If we don't want to do this, can we then rename this metric to `raw_sum_episode_return` to be consistent with the naming of the other metrics in the logs? 
Keen to hear your thoughts on this as well @KaleabTessera?

Reply:
This are great points @RuanJohn!

I am honestly worried about the confusion between these params e.g. Mean Sum Episode Return might be confusing. I think `raw_sum_episode_return` is a good compromise. We should update the logging docs to reflect this.

Reply:
Okay I will just change the name to `raw_sum_episode_return`

Reply:
I agree with that. Thank you @OmaymaMahjoub and @KaleabTessera. I don't think we have explicit logger docs anymore after the readme got trimmed down. But definitely worth introducing in the docs again at some point.